Tests/test_content.py

```diff
@@ -383,10 +383,20 @@ def mock_run(conf_json_test_details, tests_queue, tests_settings, c, demisto_use
     # Mock recording - no mock file or playback failure.
     c = demisto_client.configure(base_url=c.api_client.configuration.host,
                                  api_key=c.api_client.configuration.api_key, verify_ssl=False)
-    run_and_record(conf_json_test_details, tests_queue, tests_settings, c, demisto_user,
-                   demisto_pass, proxy, failed_playbooks, integrations, playbook_id,
-                   succeed_playbooks, test_message, test_options, slack, circle_ci,
-                   build_number, server_url, build_name)
+    succeed = run_and_record(conf_json_test_details, tests_queue, tests_settings, c, demisto_user,
+                             demisto_pass, proxy, failed_playbooks, integrations, playbook_id,
+                             succeed_playbooks, test_message, test_options, slack, circle_ci,
+                             build_number, server_url, build_name)
+    if succeed and proxy.should_validate_playback:
+        logging_manager.info('Running another playback attempt on the new record to verify that the record is valid')
+        with run_with_mock(proxy, playbook_id) as result_holder:
+            status, _ = check_integration(c, server_url, demisto_user, demisto_pass, integrations,
+                                          playbook_id, logging_manager, test_options,
+                                          is_mock_run=True)
+            result_holder[RESULT] = status == PB_Status.COMPLETED
+        if status != PB_Status.COMPLETED:
+            logging_manager.warning(
+                'Playback on newly created record has failed, see confluence page on that matter')  # TODO add link
```

Comment:
I don't know man - I don't think it's enough to just log a warning and provide a link to how to resolve the mock-related issues. If a build is green, the developer will just ignore the warning. If it were up to me and a re-recorded playback fails - that's a failure if you ask me and the build should be red. Because otherwise, what that means for us (you and me) practically speaking is that _we_ will have another test playbook and/or integration that we now need to investigate! We should stop the developer from merging it to `master` to begin with since it will just create work for us down the line. The developer should be forced to resolve the mock issues related to the integration they are working on as they are working on it.

Reply:
oh, wait. Does it already fail the build? I just saw the [screenshot](https://user-images.githubusercontent.com/41257953/102794645-5722ba00-43b4-11eb-91b2-b0d30fb8c2af.png) you attached in the PR description.